client/app/services/group/currentGroup.service.js

```diff
@@ -14,7 +14,18 @@ export default class CurrentGroup {
     Object.assign(this, {
       value: {},
       User,
-      SessionUser
+      SessionUser,
+      map: {
```

Comment:
How is this related to the current group? These seem to be proxy methods that call down to the groupMap component. Seems unexpected to me.